[2023-07-30 15:04:58,705] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_test.get_data_postgres manual__2023-07-30T15:04:57.967263+00:00 [queued]>
[2023-07-30 15:04:58,718] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_test.get_data_postgres manual__2023-07-30T15:04:57.967263+00:00 [queued]>
[2023-07-30 15:04:58,718] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 15:04:58,718] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-07-30 15:04:58,718] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 15:04:58,736] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_data_postgres> on 2023-07-30 15:04:57.967263+00:00
[2023-07-30 15:04:58,741] {standard_task_runner.py:52} INFO - Started process 1784 to run task
[2023-07-30 15:04:58,745] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_test', 'get_data_postgres', 'manual__2023-07-30T15:04:57.967263+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag_connection.py', '--cfg-path', '/tmp/tmppcn78vo3', '--error-file', '/tmp/tmp693tyggw']
[2023-07-30 15:04:58,746] {standard_task_runner.py:80} INFO - Job 325: Subtask get_data_postgres
[2023-07-30 15:04:58,821] {task_command.py:369} INFO - Running <TaskInstance: dag_test.get_data_postgres manual__2023-07-30T15:04:57.967263+00:00 [running]> on host c58846761b0c
[2023-07-30 15:04:58,897] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_test
AIRFLOW_CTX_TASK_ID=get_data_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-07-30T15:04:57.967263+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-30T15:04:57.967263+00:00
[2023-07-30 15:04:58,898] {logging_mixin.py:115} INFO - can't connect postgresql
[2023-07-30 15:04:58,898] {logging_mixin.py:115} INFO - 'Connection' object is not subscriptable
[2023-07-30 15:04:58,898] {python.py:173} INFO - Done. Returned value was: None
[2023-07-30 15:04:58,910] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_test, task_id=get_data_postgres, execution_date=20230730T150457, start_date=20230730T150458, end_date=20230730T150458
[2023-07-30 15:04:58,961] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-30 15:04:59,011] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
